swap_meet/vendor.py

```diff
@@ -0,0 +1,75 @@
+class Vendor:
+
+    def __init__(self, inventory=[]):
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_category(self, category=" "):
+        list_items = []
+        for item in self.inventory:
+            if item.category == category:
+                list_items.append(item)
+        return list_items
+
+    def swap_items(self, other, my_item, their_item):
+        if my_item not in self.inventory or their_item not in other.inventory:
+            return False
+
+        self.inventory.remove(my_item)
+        other.inventory.append(my_item)
+
+        self.inventory.append(their_item)
+        other.inventory.remove(their_item)
+
+        return True
+
+    def swap_first_item(self, other):
+
+        if len(self.inventory) < 1 or len(other.inventory) < 1:
+            return False
+
+        my_item = self.inventory[0]
+        their_item = other.inventory[0]
+
+        self.inventory.remove(my_item)
+        other.inventory.append(my_item)
+
+        self.inventory.append(their_item)
+        other.inventory.remove(their_item)
+        return True
+
+        # self.swap_items(other, my_item, their_item)
```

Comment:
this is a great idea! it would save you a LOT of unnecessary code you've already made!